docs/advanced_topics/deploying.md

```diff
@@ -60,7 +60,7 @@ The django-storages Amazon S3 backends (`storages.backends.s3boto.S3BotoStorage`
 
 ### Cache
 
-Wagtail is designed to make huge advantage of Django's [cache framework](inv:django#topics/cache) when available to accelerate page loads. The cache is especially useful for the Wagtail admin, which can't take advantage of conventional CDN caching.
+Wagtail is designed to take huge advantage of Django's [cache framework](inv:django#topics/cache) when available to accelerate page loads. The cache is especially useful for the Wagtail admin, which can't take advantage of conventional CDN caching.
```

Comment:
This sentence feels a bit wordy, maybe a simpler wording is better. 

```suggestion
Wagtail is designed to leverage Django's [cache framework](inv:django#topics/cache) when available to accelerate page loads. The cache is especially useful for the Wagtail admin, which can't take advantage of conventional CDN caching.
```

Reply:
Thank you for your feedback @lb- 
I understand the documentation better now and I'll effect the changes as soon as possible.

Reply:
@lb- @Buraah The Google developer documentation style guide discourages using leverage in this instance.

https://developers.google.com/style/word-list#leverage

> Avoid using if you mean use. If possible, use a more precise term. For example, use, build on, or take advantage of.

---

Maybe just drop the `huge`?

> Wagtail is designed to take advantage of Django's cache framework